### DISCUSSION TITLE
Dark Mode partially broken with markdown editor

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#22212] [Accessibility]
<p>When editing a Discussion Post (or an issue/PR comment? Didn’t test it on those) does the Dark theme break partially.<br>
The area with the edit options (bold, italic, Write and Preview tab, etc) will be in a blue tone, while your name will remain with a dark grey color, which looks extremely out of place and is made worse by the fact, that your avatar is partially within the blue area.</p>
<p>Example image:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/f/0f5fb7b93110171b59fb2bc270026cef22ca61f1.png" data-download-href="/uploads/short-url/2c0eE1lSzGyHzZOUSkCPq1f9DUZ.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/11576465/181069556-3d57cb73-08fa-4c29-9150-2cff1c632367.png" alt="image" data-base62-sha1="2c0eE1lSzGyHzZOUSkCPq1f9DUZ" width="690" height="177" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f5fb7b93110171b59fb2bc270026cef22ca61f1_2_690x177.png, https://aws1.discourse-cdn.com/github/original/3X/0/f/0f5fb7b93110171b59fb2bc270026cef22ca61f1.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/f/0f5fb7b93110171b59fb2bc270026cef22ca61f1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/f/0f5fb7b93110171b59fb2bc270026cef22ca61f1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">936×241 11.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks <a class="mention" href="/Andre601">@Andre601</a> for catching and reporting this! I’m able to replicate this as well and had reported a similar issue for the pointers in Issue comments. I’ve added this to our backlog to investigate :pray:</p>

**Comment by cehfisher** [comment]
Hi @Andre601 @queenofcorgis - this stale issue seems to have been resolved. Could you please confirm and close this issue?
